apps/src/templates/sectionProgress/SummaryViewLegend.jsx

```diff
@@ -25,16 +25,21 @@ const styles = {
   },
   boxStyle: {
     margin: '0 auto'
+  },
+  completedBoxes: {
+    display: 'flex',
+    flexDirection: 'row'
```

Comment:
i _think_ we can remove `flexDirection` here because 'row' is the default direction

Reply:
I tried without this first and it was staked in a column :(